Dockerfile.bak

```diff
@@ -0,0 +1,79 @@
+ARG BASE_IMAGE=python:3.7-slim
```

Comment:
delete this file

Reply:
Yes of course. This is why the PR is still in draft mode.